stellar_model/response/effect_response.py

```diff
@@ -0,0 +1,36 @@
+from pydantic import BaseModel
+
+from stellar_model import __issues__
+from stellar_model.model.horizon.effects import _EFFECT_TYPE_I_MAP
+from stellar_model.model.horizon.effects import _EFFECT_TYPE_UNION
+
+
+__all__ = ["EffectResponse"]
+
+
+class EffectResponse(BaseModel):
+    """
+    Represents single effect response.
+    Can be used for the following endpoint(s):
+        - GET /effects/:effect_id
```

Comment:
Hi, Horizon doesn't seem to contain such an API endpoint, if it exists, can you provide a URL that only contains one effect record?